pkg/helm/client_test.go

```diff
@@ -102,11 +102,11 @@ func Test_cleanUpGenericMap(t *testing.T) {
 			want: map[string]interface{}{
 				"nest": map[string]interface{}{
 					"abc":    "xyz",
-					"number": 5,
+					"number": float64(5),
 					"float":  1.5,
 					"bool":   true,
-					"array": []map[string]interface{}{
-						{
+					"array": []interface{}{
```

Comment:
think helm will accept an `[]interface{}`?

Reply:
i assume since its passing tests...

Reply:
our e2e tests never caught this bug, which is why these unit tests were introduced. so i wouldn't be so confident...

Reply:
if `want` did not change at all in these unit tests, i would be more confident in the change.

Reply:
i added comments to clarify